app/views/landmarks/show.html.erb

```diff
@@ -1,9 +1,9 @@
 <h3><%= @landmark.name %></h3>
-<%= @landmark.user.email %>
```

Comment:
@Salisbum That's your choice, but I would argue that styling each feature is part of core functionality. </two-cents>